pywbemtools/pywbemcli/_context_obj.py

```diff
@@ -402,7 +405,8 @@ def connect_wbem_server(self):
                 log=self.log,
                 use_pull=self.use_pull,
                 timestats=self.timestats,
-                verbose=self.verbose)
+                verbose=self.verbose,
+                warn=self.warn)
```

Comment:
First, there is a second call to `create_connection()` in this module. But foremost: The `create_connection()` method does not accept a `warn` parameter.

Reply:
Done.   My error removing the first attempt to fix this by passing warn around.